org.sn.myutils.pubsub/src/main/java/org/sn/myutils/pubsub/DistributedMessageServer.java

```diff
@@ -600,6 +601,7 @@ public void saveMessage(PublishMessage publishMessage, Consumer<SubscriberParams
          * @param clientMachine the client to send messages to
          * @param topics null means send messages for all topics, otherwise send messages only for these topics
          * @param minClientTimestamp find messages on or after this client timestamp
+         * @param maxClientTimestamp find messages on or before this client timestamp
          * @param lowerBoundInclusive send messages from this point. If null, calculate lowerBoundInclusive as the current server index the client is on plus one.
          * @param upperBoundInclusive send messages till this point
```

Comment:
Javadoc that find by myself n client timestamp may miss as items sorted by server index, and use binary search may not find